source/cServerData.cpp

```diff
@@ -6565,6 +6616,15 @@ auto CServerData::HandleLine( const std::string& tag, const std::string& value )
 		case 349:	 // LOOTDECAYSWITHNPCCORPSE
 			NpcCorpseLootDecay( static_cast<UI16>( std::stoul( value, nullptr, 0 )) != 0 );
 			break;
+		case 350:	// HEALTHREGENCAP
+			HealthRegenCap( std::stof( value ));
```

Comment:
stof returns a floating point value - but the caps take a SI16.

I'd expect this to be different e.g. stoi